react/components/ProductImages/components/Carousel/ThumbnailSwiper.js

```diff
@@ -30,25 +31,27 @@ const Thumbnail = ({
       onClick={() => gallerySwiper && gallerySwiper.slideTo(index)}
     >
       <figure
-        className={styles.figure}
+        className={handles.figure}
         itemProp="associatedMedia"
         itemScope
         itemType="http://schema.org/ImageObject"
       >
         <img
-          className="w-100 h-auto db"
+          className={`${handles.thumbImg} w-100 h-auto db`}
           itemProp="thumbnail"
           alt={alt}
           src={imageUrl(thumbUrl, THUMB_SIZE, THUMB_MAX_SIZE, aspectRatio)}
         />
       </figure>
       <div
-        className={`absolute absolute--fill b--solid b--muted-2 bw1 ${styles.carouselThumbBorder}`}
+        className={`absolute absolute--fill b--solid b--muted-2 bw0 ${handles.carouselThumbBorder}`}
```

Comment:
The value of `bw1`was being overwritten in the `styles .css` file and it was added `border-width: 0`, so I changed the tachyons class to be this one